README.md

```diff
@@ -20,9 +20,19 @@ Once Helm is set up properly, add the repo as follows:
 helm repo add trino https://trinodb.github.io/charts/
 ```
 
-You can then run `helm search repo trino` to see the charts.
+Run `helm search repo trino` to see the latest charts in the `trino` repo with
+an output similar to the following:
 
-Then you can install chart using:
+```
+NAME               	CHART VERSION	APP VERSION	DESCRIPTION
+trino/trino        	0.34.0       	465        	Fast distributed SQL query engine for big data ...
```

Comment:
I think we should update this with the next release since we will change to. `1.x` but then afterwards its fine if its not up to date.